File: app/src/main/java/in/woloo/www/networksUtils/NetworkAPIResponseCallback.java
package in.woloo.www.networksUtils;
import com.android.volley.VolleyError;

import org.json.JSONObject;

public interface NetworkAPIResponseCallback extends NetworkTimeOut,InternetConnection{
    void onSuccessResponse(JSONObject response, NetworkAPICallModel networkAPICallModel);
    void onFailure(VolleyError volleyError, NetworkAPICallModel networkAPICallModel);
}

File: app/src/main/java/in/woloo/www/application_kotlin/api_classes/NetworkAPIResponseCallback.kt
package `in`.woloo.www.application_kotlin.api_classes

import com.android.volley.VolleyError
import org.json.JSONObject

interface NetworkAPIResponseCallback : NetworkTimeOut, InternetConnection {
    fun onSuccessResponse(response: JSONObject?, networkAPICallModel: NetworkAPICallModel?)
    fun onFailure(volleyError: VolleyError?, networkAPICallModel: NetworkAPICallModel?)
}
